nhais-adaptor/incoming.py

```diff
@@ -5,6 +5,7 @@
 from PyInquirer import prompt
 from utilities.file_utilities import FileUtilities
 
+from adaptor.incoming.incoming_adaptor import IncomingAdaptor
 from adaptor.incoming.operation_definition_adaptor import OperationDefinitionAdaptor

```

Comment:
Think this import is no longer required.

Reply:
👍 removed